pkgs/applications/virtualization/qemu/default.nix

```diff
@@ -97,6 +97,16 @@ stdenv.mkDerivation rec {
       url = "https://gitlab.com/qemu-project/qemu/-/commit/13b250b12ad3c59114a6a17d59caf073ce45b33a.patch";
       sha256 = "0lkzfc7gdlvj4rz9wk07fskidaqysmx8911g914ds1jnczgk71mf";
     })
+    (fetchpatch {
+      name = "fix-crash-in-v9fs_walk.patch"; # remove with next release
+      url = "https://gitlab.com/qemu-project/qemu/-/commit/f83df00900816476cca41bb536e4d532b297d76e.patch";
+      sha256 = "sha256-LYGbBLS5YVgq8Bf7NVk7HBFxXq34NmZRPCEG79JPwk8=";
+    })
+    (fetchpatch {
+      name = "fix-aio-discard-return-value.patch"; # remove with next release
+      url = "https://gitlab.com/qemu-project/qemu/-/commit/13a028336f2c05e7ff47dfdaf30dfac7f4883e80.patch";
```

Comment:
I am the author of this QEMU patch, and hereby confirm that this is the correct version of the AIO discard bugfix patch, which now resides in upstream QEMU master branch.